core/lib/script-helpers.js

```diff
@@ -25,7 +27,141 @@ function getRequestForScript(networkRecords, script) {
   return networkRequest;
 }
 
+/**
+ * Estimates the number of bytes this network record would have consumed on the network based on the
+ * uncompressed size (totalBytes). Uses the actual transfer size from the network record if applicable.
+ *
+ * @param {LH.Artifacts.NetworkRequest|undefined} networkRecord
+ * @param {number} totalBytes Uncompressed size of the resource
+ * @param {LH.Crdp.Network.ResourceType=} resourceType
+ * @return {number}
+ */
+function estimateTransferSize(networkRecord, totalBytes, resourceType) {
```

Comment:
Moving these helpers elsewhere seems reasonable but `script-helpers.js` doesn't seem like the right place. These functions can operate on any arbitrary network request not just scripts.

Maybe a better location would be static functions on `NetworkRequest`? There are already some functions like that.